src/databricks/labs/remorph/install.py

```diff
@@ -261,8 +261,9 @@ def run(
         module: str,
         config: RemorphConfigs | None = None,
     ) -> RemorphConfigs:
-        self.install_rct()
-        self.install_morpheus()
+        if module in {"transpiler", "all"}:
```

Comment:
I covered during the hotfix, I have handled it differently.

https://github.com/databrickslabs/remorph/pull/1476

Reply:
Not sure re-running install tasks during config is good ? Do you not prefer the fix in this PR ?